src/foam/nanos/u2/navigation/SignUp.js

```diff
@@ -257,25 +154,21 @@ foam.CLASS({
   actions: [
     {
       name: 'login',
-      label: 'Get Started',
+      label: 'Create account',
       isEnabled: function(errors_, isLoading_) {
         return ! errors_ && ! isLoading_;
       },
       code: function(x) {
         this.isLoading_ = true;
         this.dao_
           .put(this.User.create({
-            firstName: this.firstName,
-            lastName: this.lastName,
-            organization: this.organization,
+            // organization needs to be overwritten by crunch when registering business
+            organization: this.userName,
```

Comment:
Why set it if we aren't asking for it in the form?

Reply:
Organization name is required for createBusinessDAO decorator on localUserDAO.

Reply:
There's no createBusinessDAO in foam, but I know of it lol. I don't think this is appropriate though. We are arbitrarily setting a value on the user to satisfy validation. Also I don't see organization being used anywhere in the createBusinessDAO - the only thing that may relate to it is the businessPermissionId method which uses businessName - which is another can of worms we need to address.